lib/fittings/error_handler.js

```diff
@@ -3,8 +3,8 @@
 var util = require('util');
 var http = require('http');
 
-module.exports = function create() {
-    var injector = require('../../index.js').injector;
+module.exports = function create(_, __, injector) {
```

Comment:
I'm using the underscores as placeholders for unused arguments that get passed into this function by bagpipes. I need to be able to pass in the injector during unit tests somehow, but also need the code to treat injector as undefined in production. The general convention I've seen is to use underscores for unused function arguments, though I agree the namespace collision with lodash is bad, so I've switched it to double and triple underscore.